runtime/common/src/lib.rs

```diff
@@ -158,6 +158,35 @@ impl<T: pallet_session::Config>
 	fn on_disabled(_: usize) { }
 }
 
+/// A placeholder since there is currently no provided session key handler for parachain validator
```

Comment:
can be replaced by `SessionInfo` I guess?

Reply:
Not yet everywhere, but in the long term. This struct is used in the polkadot, kusama, westend runtimes where we don't yet enable parachain runtime modules.